packages/@aws-cdk/aws-cognito/lib/user-pool.ts

```diff
@@ -509,6 +550,13 @@ export interface UserPoolProps {
    * @default true
    */
   readonly signInCaseSensitive?: boolean;
+
+  /**
+   * How will a user be able to recover their account?
+   *
+   * @default AccountRecoverySettings.PHONE_AND_EMAIL
```

Comment:
Let's not default to an option that is not recommended.

Reply:
Not recommended, but it's the default behavior on the console.

And it's also a default behavior on the cloudformation.

```
In the absence of this setting, Cognito uses the legacy behavior to determine the recovery method where SMS is preferred over email.
```

[AWS::Cognito::UserPool AccountRecoverySetting](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cognito-userpool-accountrecoverysetting.html)

Reply:
Alternatively, I could make it an essential property. But it would break the backward compatibility.

Reply:
> Not recommended, but it's the default behavior on the console.
>
> And it's also a default behavior on the cloudformation.

The CDK can have its own default, separate from CloudFormation or the console.